### PR TITLE
Adding basic service for api communication

### PR DESCRIPTION
**Pull request by wytrych**

### CODE REVIEW

**Comment by wytrych** [comment]
@incuna/js @incuna/frontend Please review

File: api-description/forms/get.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
             },
             {
                 "name": "Another form",
-                "url": "http::localhost:8000/forms/2"
+                "url": "http://localhost:8000/forms/2"
```
Comment (grahamgilchrist) [inline api-description/forms/get.json]: `://`

Comment (grahamgilchrist) [inline api-description/forms/get.json]: ah ignore me

**Comment by grahamgilchrist** [comment]
Looks good. On a side note, I'm wondering if we should deprecate the `project_settings` module pattern and instead use properly documented providers to set config values?

It slightly bothers me relying on something outside of packages which is not a dependency or versioned in any way :)

**Comment by wytrych** [comment]
Yeah I can change that and make a module instead

**Comment by grahamgilchrist** [comment]
It's unrelated to this so happy to merge, I was just thinking since we are starting this as a relatively new project, it's a good opportunity to improve how we handle config in packages.

We can do it as another PR/issue though.

**Pull request merged**